frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -288,20 +288,20 @@ export class MineTailingsInfo extends Component {
                             doc
                           )
                         }
-                      >
-                        <img className="padding-small" src={BRAND_PENCIL} alt="Edit TSF Report" />
-                      </Button>
-                      <Popconfirm
-                        placement="topLeft"
-                        title={`Are you sure you want to delete ${doc.exp_document_name}?`}
-                        onConfirm={(event) => this.removeReport(event, doc.exp_document_guid)}
-                        okText="Delete"
-                        cancelText="Cancel"
-                      >
-                        <Button ghost type="primary">
-                          <Icon type="minus-circle" theme="outlined" />
-                        </Button>
-                      </Popconfirm>
+                        string={<img src={BRAND_PENCIL} alt="Edit TSF Report" />}
+                      />
+                      <ConditionalButton
+                        popConfirm={{
+                          placement: "topLeft",
+                          title: `Are you sure you want to delete ${doc.exp_document_name}?`,
+                          onConfirm: (event) => this.removeReport(event, doc.exp_document_guid),
+                          okText: "Delete",
+                          cancelText: "Cancel",
+                        }}
```

Comment:
this should only need `{}` one set not  two `{{}}`

Reply:
Thos does no compile if I only have 1 curly brace